### ISSUE TITLE
csv source files

### ISSUE THREAD

**Issue by fsonmez** [ZhangZhiHu/assetPricing#1]
Great share for beginners like me, many thanks. Is it possible to share the source csv files used in this codes. Or, just the template of the csv files.

**Comment by vaghawan** [comment]
Yes it would have been lot helpful, if the sample used csv were here.

**Comment by alae77** [comment]
Hello guys, Did someone find the author's csv file or do you have any idea about the template ? I really need it for my PS Model replication! Thank you

**Comment by vaghawan** [comment]
Well for me, I simply opted this out and worked on my own solution.

**Comment by alae77** [comment]
If you can share your solution for PS model that would be very helpful for my thesis work. Thanks for your answer.

**Comment by vaghawan** [comment]
Well, mine is very coarse, it's just using the ordinary least square to estimate the beta using CAPM. No sophistication or anything, just one notebook.
